src/Sylius/Component/Shipping/Model/Shipment.php

```diff
@@ -22,6 +23,8 @@
  */
 class Shipment implements ShipmentInterface, TimestampableInterface
```

Comment:
But backend uses Core, and Core extends that model so your argument here is invalid :)

I guess such change should go to __different__ PR to not create monsterous one (once again :wink: ), @pjedrzejewski thoughts?